src/main/java/no/digipost/api/client/util/DigipostApiMock.java

```diff
@@ -230,27 +251,46 @@ private int servicePrintKey(HttpServerExchange httpContext, ByteArrayOutputStrea
     }
 
     private int serviceMultipartrequest(HttpServerExchange httpContext, ByteArrayOutputStream bao) throws IOException {
-        String multipart = new String(IOUtils.toByteArray(httpContext.getInputStream()));
-        String messageString = getMessageFromXMLString(multipart);
-
-        ByteArrayInputStream messageStream = new ByteArrayInputStream(messageString.getBytes());
+        HeaderMap requestHeaders = httpContext.getRequestHeaders();
+        String digipostMultipartContentType = requestHeaders.getFirst(CONTENT_TYPE);
+        String boundary = extractQuotedValueFromHeader(digipostMultipartContentType, "boundary");
+        requestHeaders.addFirst(CONTENT_TYPE, MultiPartParserDefinition.MULTIPART_FORM_DATA + "; boundary=" + boundary);
```

Comment:
Dette er litt rallete. For at parseren til undertow skal kicke inn forventer den `multipart/form-data`, og ikke vår custom digipost mediatype.